support/common/src/main/kotlin/com/studentcenter/weave/support/common/uuid/UuidCreator.kt

```diff
@@ -0,0 +1,22 @@
+package com.studentcenter.weave.support.common.uuid
+
+import java.util.*
+import com.github.f4b6a3.uuid.UuidCreator
+
+object UuidCreator {
+
+    /**
+     * uuid version 7 기반으로 uuid 를 생성합니다.
+     * millisecond 단위로 정렬된 uuid 를 생성하며,
+     * 동일한 시간에 생성될 경우 단조 증가 counter를 활용해, unique 하게 생성됩니다.
+     *
+     * @return 생성된 uuid
```

Comment:
p5) 별도로 생성되었다는 표현이 필요한지는 잘 모르곘어용ㅋㅋ
```suggestion
     * @return uuid
```